add-ons/dspace-docker-tools/ingestAIP.sh

```diff
@@ -2,6 +2,8 @@
 # Once this file has been saved to a docker volume, the ingest step will not be re-run
 CHECKFILE=/dspace/assetstore/ingest.hasrun.flag
 
+env | egrep "__.*=" | egrep -v "=.*__" | sed -e s/__/./g > /dspace/config/local.cfg
```

Comment:
I'm not so happy with overwriting local.cfg. I put a lot of time and was finally able to use Apache Commons Configuration to load enviroment variables and transform their names from containing `__` into `.`. While doing so I decided to transform capital letters to small letters as well. An environment variable DSPACE__NAME would be used as configuration property dspace.name. See:
* https://github.com/DSpace/DSpace/pull/2373
* https://jira.duraspace.org/browse/DS-4189

Unfortunately my solution works for DSpace 7 only as it requires Apache Commons Configuration 2. DSpace 6 and before uses version 1.